i18n/nl.yaml

```
@@ -0,0 +1,93 @@
+# General
+- id: read_more
+  translation: Meer lezen…
+
+- id: menu_label
+  translation: Menu
+
+- id: sidemenu_title
+  translation: Menu
+
+# Post meta
+- id: meta_lastmod
+  translation: Laatst gewijzigd
+
+- id: meta_translations
+  translation: Vertalingen
+
+# Table of Contents
+- id: toc_title
+  translation: Inhoud
+
+# Post nav
+- id: post_nav_prev
+  translation: Vorige
+
+- id: post_nav_next
+  translation: Volgende
+
+# Authorbox
+- id: authorbox_name
+  translation: Over {{ .Count }}
+
+# Sidebar
+- id: sidebar_warning
+  translation: WAARSCHUWING
+
+- id: sidebar_recommendation
+  translation: Activeer minstens één sidebar widget.
+
+# Search widget
+- id: search_placeholder
+  translation: ZOEKEN...
+
+# Languages widget
+- id: languages_title
+  translation: Talen
+
+# Categories widget
+- id: categories_title
+  translation: Categorieën
+
+# Recent Posts widget
+- id: recent_title
+  translation: Recente posts
+
+# Social widget
+- id: social_title
+  translation: Social
+
+# Tags List widget
+- id: tags_title
+  translation: Tags
+
+# Footer
+- id: footer_credits
+  translation:
+    "Gegenereerd met <a href=\"https://gohugo.io/\" rel=\"nofollow noopener\" target=\"_blank\">Hugo</a> en het \
+    <a href=\"https://github.com/Vimux/Mainroad/\" rel=\"nofollow noopener\" target=\"_blank\">Mainroad</a> theme."
+
+# 404
+- id: 404_title
+  translation: 404. Pagina niet gevonden
+
+- id: 404_text
+  translation: 'De pagina is mogelijk verplaatst, verwijderd of bestaat niet. Gebruik de zoekfunctie of "ga naar"'
```

```suggestion
  translation: "De pagina is mogelijk verplaatst, verwijderd of bestaat niet. Gebruik de zoekfunctie of \"ga naar\""
```

More consistent with other strings.

---

Done :)
(or would you like a new branch?)